drivers/SmartThings/matter-sensor/src/init.lua

```diff
@@ -20,8 +20,19 @@ local utils = require "st.utils"
 
 local BATTERY_CHECKED = "__battery_checked"
 
+local HUE_MANUFACTURER_ID = 0x100B
+
+local function supports_battery_percentage_remaining(device)
+  local battery_eps = device:get_endpoints(clusters.PowerSource.ID,
+          {feature_bitmap = clusters.PowerSource.types.PowerSourceFeature.BATTERY})
+  -- Hue devices support the PowerSource cluster but don't support reporting battery percentage remaining
```

Comment:
Do you have any thoughts about checking this through checking the feature map or supported attributes, if possible, as opposed to checks against manufacturers?

Reply:
Thanks for bringing this up. We should do exactly as you suggested and then we can remove the manufacturer specific test. This was a short term solution to get things fixed for Hue since we saw the issues there and will follow up with the better long term solution.

Reply:
Created https://smartthings.atlassian.net/browse/CHAD-11842